alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessor.java

```diff
@@ -101,7 +101,7 @@ public StaticJobAccessor(
         DistributionJobRepository distributionJobRepository,
         BlackDuckJobDetailsAccessor blackDuckJobDetailsAccessor,
         AzureBoardsJobDetailsAccessor azureBoardsJobDetailsAccessor,
-        EmailJobDetailsAccessor emailJobDetailsAccessor,
+        DefaultEmailJobDetailsAccessor emailJobDetailsAccessor,
```

Comment:
Any reason we are injecting the implementation instead of the interface? I imagine we would want to inject the most basic form of the parameter

Reply:
The interface is creating a contract that `DistributionEventReceiver` requires whereas `DefaultEmailJobDetailsAccessor` is the "whole" accessor. I suspect this will be an area that would be more clear if/when we separate things into microservice-like sub-projects.